pkg/store/store.go

```diff
@@ -0,0 +1,52 @@
+package store
+
+import (
+	"context"
+	"database/sql"
+	"log"
+)
+
+const (
+	listUsersQuery  = "SELECT * FROM Users"
+	createUserQuery = "INSERT INTO Users (name) VALUES ( ? );"
+)
+
+type Store struct {
+	db *sql.DB
+}
+
+func New(db *sql.DB) *Store {
+	return &Store{
+		db: db,
+	}
+}
+
+type User struct {
+	ID   int
+	Name string
+}
+
+func (s *Store) ListUsers(ctx context.Context) ([]User, error) {
+	rows, err := s.db.QueryContext(ctx, listUsersQuery)
+	if err != nil {
+		log.Printf("Failed to run list users query: %v", err)
+		return nil, err
+	}
+
+	users := make([]User, 0)
```

Comment:
```suggestion
	var users []User
```
nit: this is a bit shorter and functionally equivalent